app/src/main/java/com/android/streetworkapp/ui/profile/BluetoothButton.kt

```diff
@@ -0,0 +1,30 @@
+package com.android.streetworkapp.ui.profile
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.streetworkapp.device.bluetooth.BluetoothServer
+
+@Composable
+fun BluetoothButton(
+    bluetoothController: BluetoothServer,
+    uid: String // UID to send
+) {
+  val context = LocalContext.current
+
+  Button(
+      onClick = {
+        bluetoothController.startGattServer(uid) // Start GATT server to send UID
+        Toast.makeText(context, "Broadcasting UID via BLE GATT server", Toast.LENGTH_SHORT).show()
+      },
+      modifier = Modifier.size(220.dp, 50.dp).testTag("BluetoothButton")) {
+        Text(text = "Broadcast UID", fontSize = 17.sp)
```

Comment:
Suggestion: On successfully adding a friend, we could increase the score of each participant with the increase score function of the MVVM (note: this might be out of scope for this PR, you might want to do it in a future PR)

Reply:
In second PR I will do the UI but I can also integrate the `score` update. Since I want to create a small pop-up dialog. So that the client can accept or refuse the friend request from the server. What do you think ?

Cf. [dialog documentation](https://developer.android.com/develop/ui/compose/components/dialog)

Reply:
I think it is the best way to do it, you are right!